Core/Core/Networking/Connection.cs

```diff
@@ -23,12 +24,14 @@ public abstract class Connection : BackgroundService, IConnection
         private readonly ILogger _logger;
         private readonly PluginExecutor _pluginExecutor;
         private TcpClient _client;
+        private readonly ConcurrentQueue<object> _packetsToSend = new();
```

Comment:
Impossible because I'm using static interface members.

```
Connection.cs(27,63): Error CS8920 : The interface 'IPacketSerializable' cannot be used as type argument. Static member 'IPacketSerializable.Deserialize<T>(ReadOnlySpan<byte>, in int)' does not have a most specific implementation in the interface.
```

This is currently a limitation. I hope this will be fixed in C#12

Reply:
I wans't aware of that limitation. Let's hope it does get adjusted